decoder.go

```diff
@@ -58,6 +61,26 @@ func (vc videoCodec) String() string {
 	}
 }
 
+type avFrameWrapper struct {
```

Comment:
Need this because if we can't set a finalizer on C refs. Go's runtime would try to set a finalizer on a pointer that it doesn't recognize as a valid Go-allocated block and would fail